xdist/scheduler/loadscope.py

```diff
@@ -380,6 +380,10 @@ def schedule(self):
         for node in self.nodes:
             self._assign_work_unit(node)
 
+        # Ensure nodes start with at least two work units if possible (#277)
+        for node in self.nodes:
```

Comment:
while this looks correct it also may affect shutdown behaviour, as such i'd like to see how it acts in the wild